src/test/java/spoon/reflect/ast/AstCheckerTest.java

```diff
@@ -57,6 +59,22 @@
 
 public class AstCheckerTest {
 
+	@Test
+	void ctLiteralsInCtCaseExpressionShouldHaveCommentsAttached() {
```

Comment:
You could use the new `ModelTest` Annotation here. This removes all of the arrange code.